src/Common/src/Interop/Windows/Winsock/Interop.WSAIoctl.cs

```diff
@@ -32,7 +32,7 @@ internal static partial class Winsock
             [Out] byte[] outBuffer,
             [In] int outBufferSize,
             [Out] out int bytesTransferred,
-            [In] SafeHandle overlapped,
+            [In] IntPtr overlapped,
             [In] IntPtr completionRoutine);
 
         [DllImport(Interop.Libraries.Ws2_32, SetLastError = true, EntryPoint = "WSAIoctl")]
```

Comment:
SafeHandle here should probably be changed as well -- though on quick inspection, I think this method is unused and should just be removed.